cmd/fortifyExecuteScan_test.go

```diff
@@ -637,6 +647,27 @@ func TestTriggerFortifyScan(t *testing.T) {
 	})
 }
 
+func TestGetMinSpotChecksPerCategory(t *testing.T) {
+	testExpectedGetMinSpotChecksPerCategory := func(spotChecksMinUnit string, spotChecksMax int, spotChecksMin int, issuesPerCategory int, spotChecksMinCalculatedExpected int) {
+		testName := fmt.Sprintf("Test GetMinSpotChecksPerCategory for SpotCheckMinimumUnit: %v, SpotCheckMaximum: %v, SpotCheckMinimum: %v, issuesPerCategory: %v", spotChecksMinUnit, spotChecksMax, spotChecksMin, issuesPerCategory)
+		t.Run(testName, func(t *testing.T) {
+			config := fortifyExecuteScanOptions{SpotCheckMinimumUnit: spotChecksMinUnit, SpotCheckMaximum: spotChecksMax, SpotCheckMinimum: spotChecksMin}
+			spotCheckMin := getMinSpotChecksPerCategory(config, issuesPerCategory)
+			assert.Equal(t, spotChecksMinCalculatedExpected, spotCheckMin)
+		})
+	}
+
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 100, 10)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 8, 1)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 3, 1)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 50, 10, 5)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 200, 10)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 26, 3)
+	testExpectedGetMinSpotChecksPerCategory("percentage", 10, 10, 24, 2)
```

Comment:
looks much better! let's sort the list in ascending order of numbers, though.

Reply:
more test cases could also be interesting, e.g.
10, 10, 10
and some where max is zero...